File: Sorting/bubble_sort.py
def bubble_sort(array):
    sorted = False
    
    while not sorted:
        sorted = True
        for i in range(len(array) - 1):
            if array[i] > array[i + 1]:
                swap(i, i + 1, array)
                sorted = False
    
    return array

def swap(idx1, idx2, array):
    array[idx1], array[idx2] = array[idx2], array[idx1]

File: Sorting/bubbleSort.js
const bubbleSort = (array) => {
    let sorted = false
    const arrLen = array.length
    while (!sorted) {
        sorted = true
        for (let i = 0; i < arrLen; i++) {
            if (array[i] > array[i + 1]) {
                let temp = array[i + 1]
                array[i + 1] = array[i]
                array[i] = temp
                sorted = false
            }
        }
    }
    return array
}